github/github-accessors.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The go-github AUTHORS. All rights reserved.
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
```

Comment:
You'll need to revert this change. See #823 for the reason why. Sorry about the trouble.

Reply:
I've applied this trivial change.

Reply:
@shurcooL It was my pleasure working on this :smile: